eng/Build.props

```diff
@@ -24,6 +24,7 @@
       Format:
       <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\Microsoft.Extensions.Options.5.0.0.csproj" />
     -->
+    <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\Microsoft.Win32.SystemEvents.8.0.0.csproj" />
```

Comment:
You need to add System.Drawing.Common here as well. It needs to be defined SystemEvents.